src/video/ngage/SDL_ngageevents.cpp

```diff
@@ -113,22 +113,22 @@ static SDL_Scancode ConvertScancode(_THIS, int key)
             keycode = SDLK_ASTERISK;
             break;
         case EStdKeyHash:         // Hash
-            keycode = SDLK_SLASH;
+            keycode = SDLK_HASH;
             break;
         case EStdKeyDevice0:      // Left softkey
-            keycode = SDLK_F1;
+            keycode = SDLK_LGUI;
```

Comment:
You might want to use a different keycode here - SDL 1.2 maps `EStdKeyLeftFunc` and `EStdKeyRightFunc` to `SDLK_LMETA` and `SDLK_RMETA` (the old name for these keys).

Reply:
Any reason to maintain compatibility with SDL 1.2? I'm genuinely interested.